core/base/spinnakerservice.yml

```diff
@@ -6,7 +6,7 @@ spec:
   spinnakerConfig:
     # ------- Main config section, equivalent to "~/.hal/config" from Halyard
     config:
-      version: 2.21.0            # Spinnaker version to deploy
+      version: invalid            # Spinnaker version to deploy
```

Comment:
Is it fine?

Reply:
Yeah, the idea is that you use the `core/patches/version.yml` file so that we don't have stale references all over the repo to older versions 😓